src/core/border_router/routing_manager.hpp

```diff
@@ -556,6 +559,20 @@ class RoutingManager : public InstanceLocator
      *
      */
     Dhcp6PdState GetDhcp6PdState(void) const { return mPdPrefixManager.GetState(); }
+
+    /**
+     * Sets the callback to notify when the state of request/release a prefix via DHCPv6 Prefix Delegation (PD) is
+     * changed.
```

Comment:
Sets the callback whenever the state of a prefix request or release, via the DHCPv6 Prefix Delegation (PD), changes on the Thread interface.
```suggestion
     * Sets the callback to notify when the state of request/release a prefix via DHCPv6 Prefix Delegation (PD) is
     * changed.
```